test/filters/term_test.rb

```diff
@@ -13,14 +13,24 @@ def test_invalid_filter_should_have_no_keys
     refute filter.valid?
   end
 
-  def test_valid_filter_should_have_2_and_more_keys
+  def test_invalid_filter_should_have_2_and_more_keys
     filter = Elasticquery::Filters::Term.new  a: 1, b: 2
+    refute filter.valid?
+  end
+
+  def test_valid_filter_could_have_cache_option
+    filter = Elasticquery::Filters::Term.new  a: 1, _cache: false
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.